platformcomponents/desktop/attachment-file-error.json

```diff
@@ -0,0 +1,16 @@
+{
+  "attachment-file-error": {
+    "sent": {
+      "comment": "Applied to sent & blocked file attachments",
+      "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=5533%3A11688",
+      "background": "@theme-background-primary-disabled",
+      "border": "@theme-outline-secondary-normal",
+      "iconForeground": "@theme-common-text-white",
```

Comment:
like before, 
```
"previewbox":{
     "icon": "@theme-common-text-white",
     "background": "@theme-background-accent-normal"
}
```

Reply:
Pushed commit
[Grouped icon colors under previewBox key in attachment-file-error](https://github.com/momentum-design/tokens/pull/174/commits/63371ead09c44f042075f028ef58deb9eb817850)
Changing this conversation to Resolved.